README.md

```diff
@@ -17,7 +17,8 @@
 Run a single-replica cluster on Linux (or [other platforms](https://docs.tigerbeetle.com/start/)):
 
 ```console
-$ curl -Lo tigerbeetle.zip https://linux.tigerbeetle.com && unzip tigerbeetle.zip && ./tigerbeetle version
+$ curl -Lo tigerbeetle.zip https://linux.tigerbeetle.com && unzip tigerbeetle.zip
+$ ./tigerbeetle version
```

Comment:
At the moment, `./tigerbeetle version` gets cut off, but I think it's worth putting on a line by itself, to teach the user "this is how I check if TB is operational".